.github/workflows/publish.yml

```diff
@@ -0,0 +1,53 @@
+name: Publish NPM Package
+
+# Workflow to automat publishing to NPM,
+#  - Uses the NPM_TOKEN GitHub secret
+#  - Triggers when a new 'release' is created
+#  - Executes a publish dry-run to list out files
+#  - Publishes after manual approval from admin added to 'release' environment
+#  - Builds/published with Node 16 LTS
+#
+# For more details, refer to https://github.com/amazon-connect/amazon-connect-chatjs/pull/165
+
+on:
+  workflow_dispatch:
+  release:
+    types: [created]
+
+env:
+  RELEASE_NODE_VERSION: '16.x' # https://nodejs.dev/en/about/releases
+
+jobs:
+  publish-dry-run:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.RELEASE_NODE_VERSION }}
+          registry-url: 'https://registry.npmjs.org'
+
+      - run: npm ci
+      - run: npm run release
+      - run: npm publish --dry-run
+      - run: git status
+  
+  build-and-publish:
+    needs: [publish-dry-run]
+    runs-on: ubuntu-latest
+    environment:
+      name: release
+      url: https://www.npmjs.com/package/amazon-connect-chatjs
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.RELEASE_NODE_VERSION }}
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm install
+      - run: npm run release
```

Comment:
Here also we need to change to `npm ci` ?

Can you also add steps in the ReadMe - how to do `npm publish` using this script.

Reply:
Yes, will add new revision with documentation and updated workflow